src/main/java/jdbcecomarce/Connections.java

```diff
@@ -0,0 +1,198 @@
+package jdbcecomarce;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.mysql.cj.Query;
+import com.mysql.cj.protocol.Resultset;
+import com.mysql.cj.protocol.a.ResultsetFactory;
+
+public class Connections {
+
+	final static String DB_URL = "jdbc:mysql://localhost:3306/employee";
+	final static String USER = "root";
+	final static String PASSWord = "Phanirama@1997";
+
+	public static Connection getConnetion1() throws SQLException {
+
+		Connection con = DriverManager.getConnection(DB_URL, USER, PASSWord);
+
+		System.out.println("connection sucessfull");
+		return con;
+	}
+
+	public static void createTables() throws SQLException {
+		Connection connection = getConnetion1();
+
+		Statement statement = connection.createStatement();
+
+		String query = "DROP TABLE IF EXISTS passengers";
+		String query1 = "DROP TABLE IF EXISTS bookingss";
+		statement.execute(query);
```

Comment:
is this dropping tables properly ???

Reply:
yes sir

Reply:
i changed it into proper query now it's working ,sir